packages/compass-e2e-tests/tests/collection-documents-tab.test.ts

```diff
@@ -484,6 +484,10 @@ FindIterable<Document> result = collection.find(filter);`);
   });
 
   it('can copy a document from the contextual toolbar', async function () {
+    if (process.env.COMPASS_E2E_DISABLE_CLIPBOARD_USAGE === 'true') {
+      this.skip();
```

Comment:
Do we skip this test explicitly because `runFindOperation` uses the clipboard? or for some other reason? Because I see multiple `runFindOperation` in this file and only one `skip` here. Maybe we could add a comment here. Or maybe we could do something similar directly inside the `runFindOperation` function, so anyone who uses it won't miss this use case.

Reply:
`runFindOperation` is not causing issues here. It does use `setAveValue` but I added some handling there for the cases when clipboard is not working. This particular test is testing that document can be copied to clipboard and so there is really no way for us to keep it when clipboard is not working.